subgraphs/marketplace/src/mappings/smithonia-weapons.ts

```diff
@@ -0,0 +1,9 @@
+import { Transfer } from "../../generated/TreasureMarketplace/ERC721";
+import { createErc721Collection } from "../helpers";
+import * as common from "../mapping";
```

Comment:
Any reason to not just import `handleTransfer721`? I don't know if this compiler supports tree shaking, so just importing what's used may help reduce the compile time and size for this specific Smithonia Weapons WASM module

Reply:
Just copy/pasting, lol.